faq.md

```diff
@@ -10,6 +34,44 @@ ocrd workspace --help
 ocrd workspace add --help
 ```
 
-## How do I turn off logging
+### How can I specify parameters on the command line?
+
+### How to configure logging?
+
+## OCR-D module project software
+
+### Where can I find official OCR-D module project software?
+
+### Which third-party OCR-D-compatible software exists?
+
+### Which processors are available?
+
+## Workflows and processors
+
+### How can I define workflows?
+
+### Where can I find sample workflows to experiment with?
+
+### How to handle failing workflows?
+
+### Why do some processors have multiple input or output file groups?
```

Comment:
Also: "How do I define them"

Reply:
You mean like `How do I specify multiple input/output file groups?`

Reply:
Yes